test/functional/bip68-sequence.py

```diff
@@ -371,8 +371,10 @@ def activateCSV(self):
         # activation should happen at block height 432 (3 periods)
```

Comment:
Seems like this could do with more commenting. Activation happens at the end of 3 periods since:
- BIP68 bit is not yet defined in period 1
- Signaling takes place in period 2
- BIP68 is locked in during period 3.

`getblockchaininfo` will show BIP68 as active on at block 431 (144 * 3 -1) since it's returning whether BIP68 is active for the *next* block.